CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased changes
 
+- Added Upward usage in the transaction-logger service and throwing errors for unknown cases for affected_contracts
```

Comment:
Please correctly highlight `name/fileName/pathName/typeNames/functionName...` as I already pointed that out in the previous two PRs. Don't need to mention `transaction-logger` service as this is the `changelog` about the `transaction-logger` meaning redundant info.